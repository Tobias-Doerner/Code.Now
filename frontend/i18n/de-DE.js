export default {
  appbar: {
    title: 'AirPoll App'
  },
  common: {
    home: 'Code.Now'
  },
  footer: {
    title: 'Accenture Code.Now Challenge'
  },
  formatter: {
    datetime: 'DD.MM.YY HH:mm:ss'
  },
  page: {
    error: {
      link: 'Zur√ºck zum Start',
      'other error': 'Ein Fehler ist aufgetreten!',
      'page not found': 'Seite nicht gefunden!'
    }
  },
  settings: {
    darkmode: 'Nachtmodus',
    language: 'Sprache',
    title: 'Einstellungen'
  }
}
