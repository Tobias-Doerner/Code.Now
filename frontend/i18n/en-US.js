export default {
  appbar: {
    title: 'AirPoll App'
  },
  common: {
    air_pollution: 'Air Pollution',
    error: {
      network_error: 'Open AQ Platform API is currently not responsive!'
    },
    home: '<Code.Now!>'
  },
  footer: {
    title: 'Accenture Code.Now Challenge'
  },
  formatter: {
    datetime: 'D/M/YY h:mm:ss a'
  },
  measurement: {
    label: {
      city: 'City',
      country: 'Country'
    },
    table: {
      header: {
        air_pollution: 'Air Pollution',
        city: 'City',
        country: 'Country',
        date: 'Date & Time',
        latitude: 'Latitude',
        location: 'Measuring Station',
        longitude: 'Longitude',
        measurement_unit: 'Measurement Unit',
        measurement_value: 'Particulate Matter'
      },
      title: 'Measurements of pollutants'
    }
  },
  page: {
    error: {
      link: 'Back to start',
      'other error': 'An error occurred!',
      'page not found': 'Page not found!'
    },
    index: {
      card: {
        btn: 'Go to the Source',
        text:
          'Front-end of the AirPoll App implemented with Vue.js, Nuxt.js and Vuetify.'
      }
    }
  },
  settings: {
    darkmode: 'Dark Mode',
    language: 'Language',
    title: 'Settings'
  }
}
