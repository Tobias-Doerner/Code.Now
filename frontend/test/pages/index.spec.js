import Vue from 'vue'
import Vuetify from 'vuetify'
import { createLocalVue, mount } from '@vue/test-utils'
import { setupI18n } from '@/test/i18n'
import { setupVuetify } from '@/test/vuetify'

import Index from '@/pages/index.vue'

Vue.use(Vuetify)
const localVue = createLocalVue()
const i18n = setupI18n(localVue)
const vuetify = setupVuetify(localVue)

describe('index', () => {
  it('snapshot test', () => {
    const cmp = mount(Index, {
      localVue,
      i18n,
      vuetify
    })

    expect(cmp.element).toMatchSnapshot()
  })
})
