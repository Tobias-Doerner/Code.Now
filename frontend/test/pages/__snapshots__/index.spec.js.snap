// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index snapshot test 1`] = `
<div
  class="container container--fluid"
>
  <div
    class="row justify-center"
  >
    <div
      class="mx-auto v-card v-sheet theme--light tertiary"
    >
      <div
        class="v-image v-responsive theme--light"
        style="max-width: 700px;"
      >
        <transition-stub
          mode="in-out"
          name="fade-transition"
        >
          <div
            class="v-image__image v-image__image--preload v-image__image--cover"
            style="background-position: center center;"
          />
        </transition-stub>
        <div
          class="v-responsive__content"
        />
      </div>
       
      <div
        class="v-card__text body-1"
      >
        
        Frontend der AirPoll App realisiert mit Vue.js, Nuxt.js und Vuetify.
      
      </div>
       
      <div
        class="v-card__actions"
      >
        <a
          class="v-btn v-btn--is-elevated v-btn--has-bg theme--light v-size--default primary"
          href="https://github.com/Tobias-Doerner/Code.Now"
          rel="noreferrer"
          target="_blank"
        >
          <span
            class="v-btn__content"
          >
            
          Gehe zur Quelle
        
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;
